.github/workflows/release.yaml

```diff
@@ -763,16 +763,18 @@ jobs:
         if: failure()
         uses: 8398a7/action-slack@v3
         with:
-          status: ${{ job.status }}
+          status: custom
```

Comment:
changing this to custom to condense the build status output. similar to this: https://github.com/8398a7/action-slack#custom-formats-of-your-choice